reliability-v2/config/reliability-netobserv.yaml

```diff
@@ -47,28 +54,53 @@ reliability:
       interval: 10
       tasks:
         - func delete_all_projects # clear all projects
-        - func verify_project_deletion -n 8 # verfy project deletion in 2 namespaces
-        - func new_project -n 8 # new 8 projects
+        - func verify_project_deletion -n 2 # verfy project deletion in 2 namespaces
+        - func new_project -n 2 # new 2 projects
```

Comment:
I had more scale so that we could have larger load in cluster, any reason to stick to lower amount of namespaces/workloads?

Reply:
@memodi As we talked in [slack](https://redhat-internal.slack.com/archives/D07R81A5G4R/p1732250299843239), the origin load is too big that exceeded the default 250 pod per node on the 3 worker nodes. See an example here: https://redhat-internal.slack.com/archives/C0266JJ4XM5/p1732269147674219?thread_ts=1732263213.922569&cid=C0266JJ4XM5
So I tried to reduce the load to see if it can work and also fit your load requirement.